web.go

```diff
@@ -592,7 +613,19 @@ func RunWebServer() {
 	}
 
 	logger.Info().Str("bindAddress", bindAddress).Bool("loopbackOnly", config.LocalLoopbackOnly).Msg("Starting web server")
-	if err := r.Run(bindAddress); err != nil {
+	server := &http.Server{
+		Addr:    bindAddress,
+		Handler: r,
+	}
+
+	go func() {
+		for range updateWebRouter {
+			server.Handler = setupRouter(config.TLSMode != "" && config.TLSEnforce)
```

Comment:
We should _not_ be passing the config into `setupRouter()`. See [this comment](https://github.com/jetkvm/kvm/pull/1093/changes#diff-a0a6130738465fc41d08d7b30558d8f1e63d7ac34466343db619b1d020487225R77)